app/src/myanmar/res/layout/row_header_item.xml

```diff
@@ -2,19 +2,39 @@
 <merge xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
     <include layout="@layout/bottom_view"></include>
-    <org.eyeseetea.malariacare.views.TextCard
-        android:layout_width="match_parent"
+    <LinearLayout
+        android:orientation="horizontal"
+        android:layout_width="wrap_content"
         android:layout_height="match_parent"
-        android:background="@color/transparent"
         android:gravity="left"
         android:layout_gravity="left"
         android:layout_margin="@dimen/input_edit_text_margin"
         android:paddingLeft="@dimen/input_edit_text_left_padding"
-        android:paddingRight="@dimen/input_edit_text_right_padding"
-        android:textColor="@color/dark_violet"
-        android:id="@+id/row_header_text"
-        android:textSize="@dimen/input_number_edit_text_phone"
-        app:tDimension="@string/font_size_level2"
-        app:tFontName="@string/condensed_font_name"
-        />
+        android:paddingRight="@dimen/input_edit_text_right_padding">
+        <org.eyeseetea.malariacare.views.TextCard
+            android:layout_width="wrap_content"
+            android:layout_height="match_parent"
+            android:background="@color/transparent"
+            android:gravity="left"
+            android:layout_gravity="left"
+            android:textColor="@color/dark_violet"
+            android:id="@+id/row_header_text"
+            android:textSize="@dimen/input_number_edit_text_phone"
+            app:tDimension="@string/font_size_level2"
+            app:tFontName="@string/condensed_font_name"
+            />
+        <org.eyeseetea.malariacare.views.TextCard
+            android:layout_width="wrap_content"
+            android:layout_height="match_parent"
+            android:layout_toRightOf="@+id/row_header_text"
+            android:visibility="visible"
+            android:layout_gravity="center_horizontal"
+            android:textColor="@color/red"
+            android:id="@+id/row_header_compulsory"
+            android:inputType="textLongMessage"
+            android:text="*"
```

Comment:
If you look at my second comment on #506 you'll see that I attached you the image from the designer. You remarked during our last meeting that the asterisk looked more like an image than like a real asterisk. Does your red asterisk look like that one? Maybe you need to change this TextCard by an Image? What do you think?